doc/tutorials/validation.md

```diff
@@ -150,6 +151,22 @@ This [mounts](/doc/tutorials/mount.md) the backend `tutorial.dump` to the mount
 The validation plugin now uses the metadata of the keys below **/tests/spec**
 to validate values before storing them in `tutorial.dump`.
 
+If we try setting the key again, we will get an error:
+
+```sh
+kdb set user:/tests/spec/test "not a number"
+# STDERR: .*Validation Syntactic.*Not a number.*
+# ERROR:  C03100
+# RET     5
+```
+
+However, if we add a key that adheres to the validation rules, it will work:
+
+```sh
+kdb set user:/tests/spec/test 42
+#> Create a new key user:/tests/spec/test with string "42"
+```
```

Comment:
At the end you need to clean up and remove all keys. Such a cleanup exists already in line 108-110, where it is probably not needed anymore.

Reply:
There is also a cleanup on line 183-184, but I still get this when running the tests:

```
225: Test did not leave mount config in the same state at is was before!
225: This means the test itself is flawed or the KDB was externally modified while the tests were running!
225: You can inspect the original mount config in /tmp/tmp.C2eOTt6xWI/mount.export.dump
225: compared to situation now in /tmp/tmp.C2eOTt6xWI/mount.check.dump
225: 
225: Other important recovery files are also in the directory /tmp/tmp.C2eOTt6xWI
225: Please remove the /tmp/tmp.C2eOTt6xWI directory yourself after you fixed the situation, I cannot do it for you
1/1 Test #225: testshell_markdown_tutorial_validation ...***Failed    3.68 sec
```

Reply:
You need to debug the issue by looking into these files and see which leftovers there are. Make sure you remove and umount everything that was mounted before.

Reply:
Cleanup added and the pipeline now passes